manifests/calico-etcd.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 layout: null
 ---
-{% helm %}
+{% helm --include-crds=false %}
```

Comment:
What changed that made this necessary?

Reply:
I was writing up an answer to that question when I realized that my solution was flawed :sweat_smile: 

The problem this whole PR is working  around is that in the Calico chart, the CRDs are stored in a toplevel directory called `crds/`, so that they can be included manually by `kdd-crds.yaml` which has this logic:

```
{{- if eq .Values.datastore "kubernetes" -}}
{{ range $path, $_ := .Files.Glob "calicocrds/kdd/*" }}
{{ $.Files.Get $path }}
---
{{- end -}}
{{- end -}}
```

This works in helm v2, and still works in helm v3, however, **the crds/ directory has special meaning in helm v3, and will be included (a second time in this case) with --include-crds is true.**

We want to make use of this special behavior in the tigera-operator chart, and I wanted to leave the CRDs there in the Calico chart but realized that was a bad idea because it would be a change in behavior. So now, this PR just moves `crds/` to `calicords/`

Reply:
What you have now, is that the right way to go?
It kind of feels like we're working against helm. Instead should we keep things in the `crd` directory and get rid of the loop in kdd-crds.yaml (and maybe also add back the `--include-crds=false` for etcd)?

Reply:
Were you saying you think it is a bad idea to change the behavior and count on the helm behavior of including the crds?

Reply:
Good points. My latest commit has helm v2 rendering the Calico chart and helm v3 rendering the operator chart. That way, code & behavior stays the same.